controllers.js

```diff
@@ -0,0 +1,7 @@
+const { fetchTopics } = require("./model");
+
+exports.getTopics = (req, res, next) => {
+  fetchTopics().then((topics) => {
+    res.status(200).send(topics);
```

Comment:
don't forget you need to send back any data from your controllers on a meaningful key, ie. .send({topics})